### ISSUE TITLE
0 to 1.0 float images not saved correctly

### ISSUE THREAD

**Issue by marisancans** [john-guo/simply-view-image-for-python-opencv-debugging#1]
Hey, I have been searching for something like this extension, great work.
There is a bug when the image is in range 0..1 floats, it is not saved correctly.
There should probably be some checking: if img.dtype = np.float32 or np.float64 then img * 255.0
![image](https://user-images.githubusercontent.com/16538903/80311341-d214e000-87e7-11ea-83d8-1bf4cc74ed53.png)

**Comment by john-guo** [comment]
Oh, the initial version is hardcode. I simply use the "imwrite" to save image not rather read the image data through the python debugger. That means your img must be a correctly image for "imwrite". Maybe a simply workaround is we can customize the save function name if the function satisfies the save function signature "new_name(filenameTosave, variableTosave)", then we can define a custom save function in python to save correctly. Do you think so?

**Comment by marisancans** [comment]
I would like to help, but can you explain me how can I run this in vs code?
I download the repo with git clone, then I open vs code, but when I run with F5 I get error:

```
> Executing task: npm run webpack <


> simply-view-image-for-python-opencv-debugging@0.0.1 webpack /home/maris/src/simply-view-image-for-python-opencv-debugging
> webpack --mode development


Insufficient number of arguments or no entry found.
Alternatively, run 'webpack(-cli) --help' for usage info.

Hash: e45b18472ca6f69d150b
Version: webpack 4.43.0
Time: 40ms
Built at: 04/26/2020 8:48:24 PM

ERROR in Entry module not found: Error: Can't resolve '.\src\extension.ts' in '/home/maris/src/simply-view-image-for-python-opencv-debugging'
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! simply-view-image-for-python-opencv-debugging@0.0.1 webpack: `webpack --mode development`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the simply-view-image-for-python-opencv-debugging@0.0.1 webpack script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/maris/.npm/_logs/2020-04-26T17_48_24_821Z-debug.log
The terminal process terminated with exit code: 2

Terminal will be reused by tasks, press any key to close it.
```

**Comment by marisancans** [comment]
Okay I fixed the problem. It was in webpack.config.js file, line 11:
replaced   `entry: '.\\src\\extension.ts' `with `entry: './src/extension.ts'`

**Comment by john-guo** [comment]
Hi, I have merged your fix to 0.0.2.
